Procfile.heroku

```diff
@@ -0,0 +1,2 @@
+web: ./manage.py runserver -p $PORT --host 0.0.0.0
+worker: ./bin/run celery worker --app=redash.worker --beat -Q queries,celery,scheduled_queries
```

Comment:
I took that file verbatim from Procfile.dev
https://github.com/getredash/redash/blob/master/Procfile.dev
Not sure if that one needs updating too